src/components/RenderHTML.js

```diff
@@ -65,11 +68,24 @@ function CodeRenderer({
     // We split wrapper and inner styles
     // "boxModelStyle" corresponds to border, margin, padding and backgroundColor
     const {boxModelStyle, otherStyle: textStyle} = splitBoxModelStyle(style);
+
+    const italic = textStyle.fontStyle === 'italic' && fontFamily.MONOSPACE_ITALIC;
+    const bold = textStyle.fontWeight === 'bold' && fontFamily.MONOSPACE_BOLD;
+    const italicBold = italic && bold && fontFamily.MONOSPACE_BOLD_ITALIC;
+
+    const font = italicBold || bold || italic || fontFamily.MONOSPACE;
+
+    const textStyleOverride = {
+        fontFamily: font,
+        fontWeight: undefined,
+        fontStyle: undefined,
+    };
+
```

Comment:
~To help in review

That is to find the fontFamily and reset `fontWeight` and `fontStyle` that break in platforms like android.

Reply:
Very stylish way to set the font! Big fan of that. 

OK, I just want to be sure I'm understanding this correctly. We change `fontWeight` when we want **bold**, and we change `fontStyle` when we want _italic_. Right?

Why do we need the `tagStyles` at all? 

Do we only override the properties that **are not** already set in `textStyle`?

Reply:
Very stylish way to set the font! Big fan of that.

OK, I just want to be sure I'm understanding this correctly. We change fontWeight when we want bold, and we change fontStyle when we want italic. Right?

Why do we need the tagStyles at all?

Do we only override the properties that are not already set in textStyle?

Sorry if you've explained this earlier 😅

Reply:
> OK, I just want to be sure I'm understanding this correctly. We change fontWeight when we want bold, and we change fontStyle when we want italic. Right?

Yeah, we are using the font-weight and font-family to know which font must be used.

Reply:
> Why do we need the tagStyles at all?

We need it to re-style the elements from `react-native-render-html`

Reply:
> Do we only override the properties that are not already set in textStyle?

No, we override properties that was defined, by default the `react-native-render-html` add a style to the elements, for example the `<strong>` has a fontWeight `bold` and in the android it break the font, and don't work correctly.

And to work correctly we get the correct font based in the fontWeight and fontFamily.

https://github.com/meliorence/react-native-render-html/blob/819e06e175b8fa664724bec9519bb50f2da33bec/src/HTMLDefaultStyles.js#L33

Reply:
Got it. Ok cool! Thanks for those clarifiers. 👍